exporters/metric/cortex/config_test.go

```diff
@@ -132,33 +117,33 @@ func TestValidate(t *testing.T) {
 			cortex.ErrTwoPasswords,
 		},
 		{
-			"Config with Proxy URL",
-			&ExampleConfigWithProxy,
-			&ExampleConfigWithProxy,
+			"Config with Custom Timeout",
+			&ExampleRemoteTimeoutConfig,
+			&ValidatedCustomTimeoutConfig,
 			nil,
```

Comment:
This is minor, so I'll leave it up to you on which to use, but I prefer to explicitly see which fields these are assigned to so I don't have to go searching for the table test struct and figure out which is input, expected, etc.

Reply:
Thanks for the feedback! I can see how the tests are clearer and easier to read when the field names are there. I'll remember to add them in my other test files.